### DISCUSSION TITLE
`BUSTUB_PAGE_SIZE` is undeclared in GradeScope's test code

### DISCUSSION THREAD

**Discussion by El-even-11** [cmu-db/bustub#424] [General]
When I uploaded the code of project 1 to GradeScope, it failed to be compiled. But it could be compiled on my local machine.

GradeScope failed becaused of `error: use of undeclared identifier 'BUSTUB_PAGE_SIZE'`.

I found that #381 renamed `PAGE_SIZE` to `BUSTUB_PAGE_SIZE`, so I tried to use `PAGE_SIZE` and the code was successfully compiled on GradeScope.

So it seems that you forget to update the new name in GradeScope's test code?

**Comment by skyzh** [comment]
In project 1 you shouldn’t use this variable.

**Reply by El-even-11** [reply]
OK. Thx🥰
